modules/perception/onboard/component/fusion_camera_detection_component.cc

```diff
@@ -729,6 +744,31 @@ int FusionCameraDetectionComponent::InternalProc(
         prefused_message->frame_->camera_frame_supplement.image_blob.get());
   }
 
+//  Determine CIPV
+  CipvOptions cipv_options;
+  if (motion_buffer_ != nullptr) {
+    if (motion_buffer_->size() == 0) {
+      AWARN << "motion_buffer_ is empty";
+      cipv_options.velocity = 5.0f;
+      cipv_options.yaw_rate = 0.0f;
+    } else {
+      cipv_options.velocity = motion_buffer_->back().velocity;
+      cipv_options.yaw_rate = motion_buffer_->back().yaw_rate;
+    }
+    ADEBUG << "[CIPV] velocity " << cipv_options.velocity
+          << ", yaw rate: " << cipv_options.yaw_rate;
+    cipv_.DetermineCipv(camera_frame.lane_objects, cipv_options,
+      &camera_frame.tracked_objects);
+
+    // // Get Drop points
+    // // motion_buffer_ = motion_service_->GetMotionBuffer();
+    // if (motion_buffer_->size() > 0) {
+    //  cipv_.CollectDrops(motion_buffer_, &camera_frame.tracked_objects);
+    // } else {
+    //   AWARN << "motion_buffer is empty";
+    // }
```

Comment:
Remove unused code if possible.

Reply:
This code will be commented out later. I added 
// TODO (techoe): Activate CollectDrops after test